### ISSUE TITLE
Cannot find module 'events' on browser

### ISSUE THREAD

**Issue by lucianmat** [ParsePlatform/Parse-SDK-JS#244]
On version 1.8.1, loading on browser got error

```
Uncaught Error: Cannot find module 'events'
```

which is traced back to EventEmitter.js

```
if (process.env.PARSE_BUILD === 'react-native') {
  module.exports = require('EventEmitter');
} else {
  module.exports = require('events').EventEmitter;
}
```  

It is a new build configuration settings, or an error ?

**Comment by peterdotjs** [comment]
@lucianmat the code on master has the fix for this issue. https://github.com/ParsePlatform/Parse-SDK-JS/pull/240 You can use 1.7.1 in the meantime. We'll create a release once we've resolved #234.

**Comment by peterdotjs** [comment]
resolved in 1.8.2
